src/main/java/com/exasol/adapter/document/documentfetcher/files/BucketFsFileLoaderFactory.java

```diff
@@ -9,9 +10,9 @@
     private static final long serialVersionUID = 3468167710047185881L;
 
     @Override
-    public FileLoader getLoader(final String filePattern, final SegmentDescription segmentDescription,
-            final ExaConnectionInformation connectionInformation) {
-        final String url = connectionInformation.getAddress();
-        return new BucketfsFileLoader(url, filePattern, segmentDescription);
+    public FileLoader getLoader(final StringFilter filePattern, final SegmentDescription segmentDescription,
+                                final ExaConnectionInformation connectionInformation) {
+        connectionInformation.getAddress();
```

Comment:
Is this redundant? Address is not assigned and used.

Reply:
oh, yes.